Samples/excel-insert-file/README.md

```diff
@@ -58,14 +58,14 @@ To run the sample you just need to sideload the manifest. The add-in web files a
 1. Download the **manifest.xml** and **SalesTemplate.xlsx** files from this sample to a folder on your computer.
```

Comment:
Don't forget to update the version history date.

Reply:
thanks fixed!